src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -166,5 +212,60 @@ describe('DrawingAnnotations', () => {
 
             expect(wrapper.find(PopupDrawingToolbar).hasClass('ba-is-drawing')).toBe(true);
         });
+
+        describe('toolbar', () => {
+            const getWrapperWithToolbar = (
```

Comment:
We should be careful about how much of the component's types we recreate in our helpers. This may not be needed if we use a single `test.each`.